### DISCUSSION TITLE
__user.name rendered as id in templating for ldap users

### DISCUSSION THREAD

**Discussion by sdx23** [grafana/grafana#38128] [Dashboards, Panels & Visualizations]
**What happened**:
Used "${__user.name}" in a text panel. Was rendered as "2" (the id of the user viewing the panel).

**What you expected to happen**:
Should have been rendered as "John Doe" (the name of the user, as correctly displayed in the users menu).

**How to reproduce it (as minimally and precisely as possible)**:
- setup grafana with ldap users
- login with an ldap user
- notice that in the user menu the name is displayed correctly
- create a dashboard with a text panel, enter "Hello ${__user.name}", save and display the panel

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.0.5
- Data source type & version: irrelevant
- OS Grafana is installed on: Debian 10
- User OS & Browser: irrelevant
- Grafana plugins: irrelevant
- Others: -

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
